src/Nox.Types.Abstractions/TypeDefinitions/NoxSimpleTypeDefinition.cs

```diff
@@ -61,6 +61,9 @@ public class NoxSimpleTypeDefinition
     [IfEquals("Type", NoxType.DateTime)]
     public DateTimeTypeOptions? DateTimeTypeOptions { get; set; }
 
+    [IfEquals("Type", NoxType.Time)]
```

Comment:
you need to regenerate the schema, use this test NoxSolutionSchemaGenerate and commit the changes

Reply:
Updated